lib/src/fonts/fonts.dart

```diff
@@ -1,12 +1,20 @@
 import 'package:flutter_test/flutter_test.dart';
-import 'package:flutter_test_goldens/src/fonts/golden_toolkit_fonts.dart' as golden_toolkit;
+import 'package:flutter_test_goldens/src/fonts/golden_toolkit_fonts.dart'
+    as golden_toolkit;
+
+/// Remember if fonts have already been loaded in this isolate.
+bool _fontsLoaded = false;
```

Comment:
Is this actually needed? What's the current behavior that you're trying to fix with this?

Reply:
Seems to me like right now if I have multiple `testGoldenScene` tests in the same suite, they will all read the font manifest, even though the fonts have already been loaded. The same thing would happen with my change, but even if we don't move font loading somewhere else it seems like wasted work?